demo.py

```diff
@@ -6,7 +6,7 @@ def func_b(id: int):
     # do some things
     return id + 55
 
-def func_a(obj: O):
+def func_a(id: int):
     # do some more things
-    b = func_b(obj)
+    b = func_b(id)
     return b
```

Comment:
should we be ending files with newlines? do we need a config for this to standardize?